### DISCUSSION TITLE
May I ask how I can obtain its APK version

### DISCUSSION THREAD

**Discussion by DpprZ** [artisan-roaster-scope/artisan#1669] [Q&A]
May I ask how I can obtain its APK version

**Comment by MAKOMO** [comment]
See https://artisan-scope.org/about/

**Reply by DpprZ** [reply]
Thank you for your reply. Unfortunately, it seems that there is no APK or IOS format file for it in this link

**Reply by MAKOMO** [reply]
This link tells you on which platforms Artisan is running. iOS and Android is NOT listed thus not supported.

**Reply by DpprZ** [reply]
All right, thanks!
